src/payment/strategies/braintree/braintree-credit-card-payment-strategy.ts

```diff
@@ -36,8 +38,14 @@ export default class BraintreeCreditCardPaymentStrategy implements PaymentStrate
 
         try {
             this._braintreePaymentProcessor.initialize(paymentMethod.clientToken, options.braintree);
-            this._is3dsEnabled = paymentMethod.config.is3dsEnabled;
 
+            if (this._isHostedPaymentFormEnabled(options.methodId, options.gatewayId) && options.braintree?.form) {
```

Comment:
if `options.braintree?.form` is not passed, should we throw a specific `InvalidArgument` exception?

Reply:
We should, but for the next major version. To make this change backward compatible, it has to be optional for now.